lxc/config.go

```diff
@@ -600,6 +600,10 @@ func (c *cmdConfigSet) Run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if server.Config == nil {
+		server.Config = map[string]interface{}{}
```

Comment:
We've switched to `any` instead of `interface{}` in most places. I think we should do the same here.

Reply:
Agreed

Reply:
fixed!